gerrit/install.sls

```diff
@@ -24,12 +24,36 @@ group_{{ settings.group }}:
     - group: {{ settings.group }}
     - makedirs: true
 
+{{ settings.base_directory }}/{{ settings.site_directory }}/lib:
+  file.directory:
+    - user: {{ settings.user }}
+    - group: {{ settings.group }}
+    - makedirs: true
+
+{% for name, library in salt['pillar.get']('gerrit:libraries', {}).items() %}
```

Comment:
Same here with the libraries, there's no example pillar so users won't know how to use these.